website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -49,6 +49,54 @@ You can read more about each of these behavior changes in the following links:
 
 - (Introduced, disabled by default) [`validate_macro_args`](/reference/global-configs/behavior-changes#macro-argument-validation). If the flag is set to `True`, dbt will raise a warning if the argument `type` names you've added in your macro YAMLs don't match the argument names in your macro or if the argument types aren't valid according to the [supported types](/reference/resource-properties/arguments#supported-types).
 
+### Deprecation warnings
+
+New deprecation warnings in v1.10.
+
+#### Duplicate keys in profiles.yml
+
+Previously, if identical keys existed in the [`profiles.yml` file](/docs/core/connect-data-platform/profiles.yml), dbt would use the last configuration listed in the file. 
+
+```yml
+
+my_profile:
+  target: 
+  outputs:
+...
+
+my_profile: # dbt would use this profile key
+  target: 
+  outputs:
+...
+
+```
+
+dbt will stop supporting duplicate keys with silent overwrite in the future. In v1.10, if two identical keys exist in the `profiles.yml`, you will get a warning, and in a future version, dbt will stop supporting this entirely. 
```

Comment:
Can we start with the deprecation warning and then give the context of how it worked historically? If someone is browsing this makes it easier/quicker to spot